test/pytest/test_metrics_kf.py

```diff
@@ -11,13 +11,14 @@
 import test_utils
 
 NUM_STARTUP_CFG = 0
-REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
-snapshot_file = os.path.join(REPO_ROOT,"test/config_kf.properties")
+REPO_ROOT = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../"))
```

Comment:
We can fix this too, maybe break apart into more than one line since it's getting hard to read

Reply:
```suggestion
REPO_ROOT = os.path.normpath(
    os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
)
```
Somthing like that? I don't see any other way